RussianFilter/sections/specific.txt

```diff
@@ -36,6 +36,8 @@ oka.fm##.rek-module
 pickpoint.ru##.bn-index-ad1
 mangalib.me##.ads
 gorlonos.com###DDWidgetSearch
+||porno-chiks.com/player/html.php?aid=pause*
+||porno-chiks.com/contents/other/player/prerol.mp4
```

Comment:
and as a substitute rule for potential use in the future I propose 
`/player/prerol.mp4$domain=porno-chiks.com`

Reply:
not sure if it popular)